src/test/java/api/loans/AgedToLostScheduledNoticesProcessingTests.java

```diff
@@ -43,6 +44,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Stream;
 
+import api.support.builders.AddInfoRequestBuilder;
```

Comment:
I think it would make sense to rename `AddInfoRequestBuilder` -> `AdditionalInfoRequestBuilder` and `AddInfoRequest` -> `AdditionalInfoRequest` to keep the naming consistent

Reply:
It is not additional information 
It is like verb add
"add information request builder" 
And AddInfoRequestBuilder is not part of this pull request